R/metadata.R

```diff
@@ -153,16 +160,17 @@ get_metadata <- function(
         cached_connection
     }
     else {
-        db_path <- file.path(cache_directory, "metadata.0.2.3.parquet")
-        
-        if (!file.exists(db_path)){
-            report_file_sizes(remote_url)
+        db_path <- file.path(cache_directory, remote_url |> names()) 
+        purrr::map2(remote_url, db_path, function(url, db_path) {
```

Comment:
Oh, I forgot to mention that Stefano's preferred style for this package is to `import` the packages using roxygen, rather than using the namespace `::` notation.